statsd/statsd.go

```diff
@@ -285,8 +285,8 @@ func (c *Client) Set(name string, value string, tags []string, rate float64) err
 }
 
 // Timing sends timing information, it is an alias for TimeInMilliseconds
-func (c *Client) Timing(name string, value float64, tags []string, rate float64) error {
-	return c.TimeInMilliseconds(name, value, tags, rate)
+func (c *Client) Timing(name string, value time.Duration, tags []string, rate float64) error {
+	return c.TimeInMilliseconds(name, float64(value.Nanoseconds())/float64(time.Millisecond), tags, rate)
```

Comment:
There's [a whole thread on this](https://groups.google.com/forum/#!topic/golang-nuts/pret7hjDc70) that I read when initially working on this patch. I'm content with whatever the library authors are content with.